crates/sui-authority-aggregation/src/lib.rs

```diff
@@ -40,7 +41,8 @@ pub async fn quorum_map_then_reduce_with_timeout_and_prefs<
     initial_state: S,
     map_each_authority: FMap,
     reduce_result: FReduce,
-    initial_timeout: Duration,
+    max_timeout: Duration,
+    min_timeout: Option<Duration>,
```

Comment:
`max_timeout` isn't used when `min_timeout` is set, is this the expected behavior?

also let's add comments on what these parameters do, their names are not really self-descriptive :)